example/docker-compose.yml

```diff
@@ -1,6 +1,8 @@
 services:
   httpbin:
     image: mccutchen/go-httpbin:v2.5.0
+    environment:
+      - MAX_BODY_SIZE=15728640 # 15 MiB
```

Comment:
Just curious, is there any reason for such number or just came up with something reasonable?

Reply:
I wanted to set something greater than the default `SecRequestBodyLimit` (`13107200`). Rounding it to 15MiB seemed to be reasonable 😅.
Proposing to add it here mainly to explicit how to change it, being a little different compared to the previous `httpbin` (which seems to not have a limitation)